Configuration/Eras/python/Era_Run2_2016_HIPM_cff.py

```diff
@@ -2,6 +2,7 @@
 
 from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
 from Configuration.Eras.Modifier_tracker_apv_vfp30_2016_cff import tracker_apv_vfp30_2016
+from Configuration.Eras.Modifier_strips_vfp30_2016_cff import strips_vfp30_2016
 
-Run2_2016_HIPM = cms.ModifierChain(Run2_2016, tracker_apv_vfp30_2016)
+Run2_2016_HIPM = cms.ModifierChain(Run2_2016, strips_vfp30_2016, tracker_apv_vfp30_2016)
```

Comment:
what is the motivation to have a separate modifier for the same purpose?
I think that we can just stay with the existing ```tracker_apv_vfp30_2016``` or if so desired, rename it to ```strips_vfp30_2016```

Reply:
Currently the `strips_vfp30_2016` modifier (included in `Run2_2016`) is the one that changes the event content, and `tracker_apv_vfp30_2016` (included in `Run2_2016_HIPM`) enables the APV simulation in the digitizer/mixing modules, and HIP mitigation in the reconstruction.
With the eventcontent customisation only in `tracker_apv_vfp30_2016` this boolean would only be there for samples/events with the APV simulation and HIP mitigation on (so it may need more care when merging the two, if needed).
What would be the preferred solution (also for the backport to 10_6)?

Reply:
I think that it should be fine to always add the new boolean to flag if the simulation is on.